cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabasePrivilegeTest.scala

```diff
@@ -40,111 +40,111 @@ class DatabasePrivilegeTest extends AdministrationCommandTestBase {
 //
 
 GRANT STOP ON DATABASE * TO my_role
 ###
 
-Grant privilege to stop all databases to the role `my_role`.
+Grant privilege to stop all databases to a role.
 
 ###assertion=update-one
 //
 
 GRANT CREATE INDEX ON DATABASE foo TO my_role
 ###
 
-Grant privilege to create indexes on the database `foo` to the role `my_role`.
+Grant privilege to create indexes on a specified database to a role.
 
 ###assertion=update-one
 //
 
 GRANT DROP INDEX ON DATABASE foo TO my_role
 ###
 
-Grant privilege to drop indexes on the database `foo` to the role `my_role`.
+Grant privilege to drop indexes on a specified database to a role.
 
 ###assertion=update-two
 //
 
 DENY INDEX MANAGEMENT ON DATABASE bar TO my_role
 ###
 
-Deny privilege to create and drop indexes on the database `bar` to the role `my_role`.
+Deny privilege to create and drop indexes on a specified database to a role.
 
 ###assertion=update-one
 //
 
 GRANT CREATE CONSTRAINT ON DATABASE * TO my_role
 ###
 
-Grant privilege to create constraints on all databases to the role `my_role`.
+Grant privilege to create constraints on all databases to a role.
 
 ###assertion=update-one
 //
 
 DENY DROP CONSTRAINT ON DATABASE * TO my_role
 ###
 
-Deny privilege to drop constraints on all databases to the role `my_role`.
+Deny privilege to drop constraints on all databases to a role.
 
 ###assertion=update-two
 //
 
 REVOKE CONSTRAINT ON DATABASE * FROM my_role
 ###
 
-Revoke existing privileges (both granted and denied) to create and drop constraints on all databases from the role `my_role`.
+Revoke granted and denied privileges to create and drop constraints on all databases from a role.
 
 ###assertion=update-one
 //
 
 GRANT CREATE NEW LABELS ON DATABASE * TO my_role
 ###
 
-Grant privilege to create new labels on all databases to the role `my_role`.
+Grant privilege to create new labels on all databases to a role.
 
 ###assertion=update-one
 //
 
 DENY CREATE NEW TYPES ON DATABASE foo TO my_role
 ###
 
-Deny privilege to create new relationship types on the database `foo` to the role `my_role`.
+Deny privilege to create new relationship types on a specified database to a role.
 
 ###assertion=update-one
 //
 
 REVOKE GRANT CREATE NEW PROPERTY NAMES ON DATABASE bar FROM my_role
 ###
 
-Revoke grant privilege to create new property keys on the database `bar` from the role `my_role`.
+Revoke the grant privilege from a role to create new property names on a specified database.
 
 ###assertion=update-two
 //
 
 GRANT NAME MANAGEMENT ON DATABASE * TO my_role
 ###
 
-Grant privilege to create labels, relationship types and property keys on all databases to the role `my_role`.
+Grant privilege to create labels, relationship types, and property names on all databases to a role.
 
 ###assertion=update-ten
 //
 
 GRANT ALL ON DATABASE baz TO my_role
 ###
 
-Grant all database privileges above on the database `baz` to the role `my_role`.
+Grant all privileges on a specified database to a role.
```

Comment:
Since this is only granting the Database privileges and not graph or dbms privileges maybe the `database` should stay?

```suggestion
Grant all database privileges on a specified database to a role.
```